src/icp_v2.cc

```diff
@@ -601,7 +601,7 @@ icpHandleIcpV2(int fd, Ip::Address &from, char *buf, int len)
         break;
 
     default:
-        debugs(12, DBG_CRITICAL, "icpHandleIcpV2: UNKNOWN OPCODE: " << header.opcode << " from " << from);
+        debugs(12, DBG_CRITICAL, "ERROR: icpHandleIcpV2: UNKNOWN OPCODE: " << header.opcode << " from " << from);
```

Comment:
can you lower case the "UNKNOWN OPCODE" text while touching these lines please.

Reply:
Done in ee36c93.